src/WindowsTracing/KrabsTracer.h

```diff
@@ -29,20 +30,33 @@ class KrabsTracer {
  private:
   void SetTraceProperties();
   void EnableProviders();
+  void EnableSystemProfilePrivilege(bool value);
```

Comment:
This method is called "Enable..." but can either enable or disable depending on the parameter (or I might be getting it wrong)? I'd rename ("Change...") or duplicate ("Enable...", "Disable...", with no parameter).

Reply:
Changed to 'SetIsSystemProfilePrivilegeEnabled'